subworkflows/local/bam_variant_calling_cnvkit/main.nf

```diff
@@ -21,6 +23,15 @@ workflow BAM_VARIANT_CALLING_CNVKIT {
 
     CNVKIT_BATCH(cram, fasta, fasta_fai, targets, reference, generate_pon)
 
+    // right now we do not use an input VCF to improve the calling of B alleles
```

Comment:
I think the readability of lines 26-29 could be improved by adding periods (.) and starting new sentences with a capital letter. (I think there are two sentences, but I'm not sure 😆 )